src/main/java/org/micks/champmaker/register/RegisterDTO.java

```diff
@@ -0,0 +1,22 @@
+package org.micks.champmaker.register;
+
+public class RegisterDTO {
```

Comment:
```suggestion
public class RegisterDto {
```
Leaving abbreviations in all capitals can lead to really bad names like ORMXMLSAMLConverter - but that would apply to all DTOs in the project 😄